Added id's to output csv of ptw-script

abhinavk99 opened the pull request:
Fixes #19

keiclone commented:
also do we need to update the readme?

abhinavk99 commented:
Yeah ill put the new csv example in there

abhinavk99 commented:
I ran `git pull` and it said it was already up to date. In both master and this, the unit test folder is under test.

keiclone commented:
then you'll need to make sure you don't create a new fixtures folder, e.g. `test/fixtures/vcr_cassettes/check_usernames/caps-username-success.yaml`

abhinavk99 commented:
Hmm I see that the fixtures are in the wrong folder, I wonder why it's messed up.

keiclone commented:
Oh I see why. It's my fault, I didn't update the vcr decorators in check_usernames

abhinavk99 commented:
I updated the decorators in check_usernames and removed the extra fixtures.

Also, I'm failing all 3 integration tests because of the error below:
```python
cls = <class 'sqlalchemy.dialects.mysql.pymysql.MySQLDialect_pymysql'>

    @classmethod
    def dbapi(cls):
>       return __import__("pymysql")
E       ModuleNotFoundError: No module named 'pymysql'
```

keiclone commented:
Oh, looks like sql alchemy does an implicit import on pymysql

keiclone commented:
Let me fix that

keiclone commented:
okay try now?

abhinavk99 commented:
I think it's because i didn't set up port forwarding. Not sure I have the time to do that today.

```bash
Error connecting to MySQL. Did you forget to set up port forwarding? 
https: // www.namecheap.com/support/knowledgebase/article.aspx/1249/89/how-to-remotely-connect-to-a-mysql-database-located-on-our-shared-server
(pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' ([WinError 10061] No connection could be made because the target machine actively refused it)") (Background on this error at: http://sqlalche.me/e/e3q8)
```

abhinavk99 commented:
It's unrelated to this PR so i think the pr can be merged.

keiclone commented on ptw-script/ptw-script.py:
```diff
@@ -47,10 +47,11 @@ def main():
     print(f'Length of list of anime: {len(anime_list)}')
 
     # Store PTW of each anime in a dict
```
minor but can you update this comment?

abhinavk99 replied:
okay i will

keiclone replied:
feel free to merge whenever

keiclone commented:
oh wait, are we updating the csv example in the README in this PR or in a different one?

abhinavk99 commented:
I'm updating it rn
